src/components/ActionPage/MiniBackoffice.tsx

```diff
@@ -1,12 +1,11 @@
 import React from "react";
 import styled from "styled-components";
 
-import useWeb3 from "../../hooks/web3";
 import { Button, Card as OriginalCard, Text } from "../ui";
 import { TextSizes } from "../ui/Text";
 import { CenteredElement, CenteredFlexElement } from "../ui/CenteredElement";
-import useMiniBackoffice from "../../hooks/miniBackoffice";
 import { unreachable } from "../../lib/types";
+import { Backoffice } from "hooks/miniBackoffice";
```

Comment:
```suggestion
import { Backoffice } from "../../hooks/miniBackoffice";
```

I think you only got away with this because TS types are erased from the output. But this is totally something I think will confuse future reader, so let's stick to using the JS way of importing names. :)

Reply:
This was left over from when I was looking into the path issue on gatsby and got left in. resolving.